variantlib/commands/generate_index_json.py

```diff
@@ -59,13 +60,17 @@ def generate_index_json(args: list[str]) -> None:
             )
             continue
 
-        if wheel_info.group("variant_hash") is None:
+        if (vhash := wheel_info.group("variant_hash")) is None:
             logger.debug(
-                "Filepath: `%(input_file)s` ... is not a wheel variant.",
+                "Filepath: `%(input_file)s` ... is not a wheel variant. Skipping ...",
                 {"input_file": wheel.name},
             )
             continue
 
+        if vhash == "0" * VARIANT_HASH_LEN:
+            known_variants[vhash] = VariantDescription()
+            continue
```

Comment:
Why do we need to special-case it here? I think it would make sense for the algorithm to be able to handle it naturally.

Reply:
Indeed. Fixed